If CXXFLAGS environment variable is set, sunpinyin will FTBFS.
temporary disable use CXXFLAGS environment variable.
--- a/SConstruct
+++ b/SConstruct
@@ -163,14 +163,6 @@
     print 'Warning: you\'ve set %s as C++ compiler' % os.environ['CXX']
     env['CXX']=os.environ['CXX']
 
-if 'CFLAGS' in os.environ:
-    print 'Warning: you\'ve set an external compiler flags for C.'
-    env['CFLAGS'] = os.environ['CFLAGS']
-
-if 'CXXFLAGS' in os.environ:
-    print 'Warning: you\'ve set an external compiler flags for C++.'
-    env['CXXFLAGS'] = os.environ['CXXFLAGS']
-
 if 'TAR' in os.environ:
     print 'Warning: you\'ve set %s as tar' % os.environ['TAR']
     env['TAR'] = os.environ['TAR']
